### PR TITLE
adapter_fix: minor cleanup

### PR DESCRIPTION
**Pull request by DanielElisenberg**
# ADAPTER FIX

* Moved FileAdapter interface to adapter folder. This seemed more intuitive to me, but if this feels messy I can move it back.
* The logic for whether the parquet is partitioned or not is minimal here, so not much need to refactor
* I have a question about the gcs_adapter (look below)
* Did some minor housekeeping

### CODE REVIEW

File: data_service/adapters/storage/gcs.py [hunk 1]
```diff
@@ -50,6 +55,7 @@ def __download_partitioned_file_from_storage(self, path: str) -> str:
 
         os.makedirs(blob_download_path)
 
+        # Is this path to partitioned parquet valid considering todays structure?
```
Comment (DanielElisenberg) [inline data_service/adapters/storage/gcs.py:58]: I'm not sure this path is valid any more? the key for the bucket doesn't match the file system nearly at all, and does not contain a version number 🤔

Comment (vaskalan) [reply data_service/adapters/storage/gcs.py:58]: Do we need to focus on this issue now? We walk through all details when we move to cloud, anyway.

Comment (DanielElisenberg) [reply data_service/adapters/storage/gcs.py:58]: We can let this module be until we migrate to cloud 👍🏻

**Review by pawbu — APPROVED** [review]
Looks good

**Pull request merged**